tsconfig.json

```diff
@@ -15,12 +15,14 @@
     "noImplicitAny": true,
     "noResolve": false,
     "paths": {
-      "*": ["./@types/*"]
+      "*": ["./@types/*", "./node_modules/*/@types/*"],
```

Comment:
I tried building this branch and these lines are causing it to fail

Reply:
interesting, I can check it out. Do you mind sharing the error output?